### DISCUSSION TITLE
***IMPORTANT*** Do not upgrade the Home Assistant go2rtc addon to v1.9.0!!!

### DISCUSSION THREAD

**Discussion by tsightler** [tsightler/ring-mqtt#827] [Announcements]
I don't really know where to put this, but it's important that users not upgrade their go2rtc addon to v1.9.0 in Home Assistant.  If you do so, and then view a video, the session will never end and the stream will stay active continuously.  The only fix I've found so far is to downgrade the go2rtc addon back to v1.8.5.

**Comment by DabblerIOT** [comment]
thank you

**Comment by tsightler** [comment]
Seems to be fixed with 1.9.1 so closing this.
